meta-leda-components/recipes-sdv/eclipse-leda/sdv-example-containers.bb

```diff
@@ -46,6 +47,8 @@ do_install:append() {
 PACKAGES = "${PN}"
 
 FILES:${PN} += "${KANTO_MANIFESTS_LOCAL_DIR}"
+FILES:${PN} += "${KANTO_MANIFESTS_LOCAL_DEV_DIR}"
+
```

Comment:
seatadjuster-example.json.disabled must be added

Reply:
That didn't exist in my branch, should automatically be fixed with the merge.